apps/communications/dialer/test/marionette/PO_test.js

```diff
@@ -0,0 +1,77 @@
+/* global require, marionette, setup, suite, test*/
+'use strict';
+
+var assert = require('chai').assert;
+
+var Dialer = require('./lib/dialer.js');
+//var CallLogView = require('./lib/views/call_log/views.js');
+var DialerOptionsView = require('./lib/views/dialer_options/views.js');
+
+marionette('Dialer > Keypad', function() {
+
+  var client = marionette.client({
+    profile: {
+      prefs: {
+        'focusmanager.testmode': true
+      }
+    },
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
+  });
+  var dialerApp;
+  var actions;
+
+  setup(function() {
+    actions = client.loader.getActions();
```

Comment:
These `actions` are the ones used to perform [many actions at the same time](http://marionette-client.readthedocs.org/en/latest/reference.html#actions). If no test uses them, there's no need to keep it.